docs/docs/addons-perf.md

```diff
@@ -14,6 +14,10 @@ next: test-utils.html
 import Perf from 'react-addons-perf' // ES6
 var Perf = require('react-addons-perf') // ES5 with npm
 var Perf = React.addons.Perf; // ES5 with react-with-addons.js
+
+// If Perf is not accessible from console i.e throws ReferenceError: 
+// Perf is not defined, create a global
```

Comment:
Question for understanding: Should it be available globally by default? When will it be available and when not? What circumstances trigger the different states?